evproc/exc.py

```diff
@@ -57,4 +60,5 @@ def __init__(self, retval=None):
 
         super(StopProcessing, self).__init__('Stop processing')
 
-        self.retval = retval
+        if retval is not _unset:
+            self.retval = retval
```

Comment:
I'd prefer setting ``retval`` unconditionally and then picking the behavior based on whether its value is ``_unset`` or not…